src/H5Fpublic.h

```diff
@@ -231,9 +231,10 @@ typedef herr_t (*H5F_flush_cb_t)(hid_t object_id, void *udata);
                *   high number of unused bits.  See documentation for                                        \
                *   H5Pset_relax_file_integrity_checks for details. */
 #define H5F_RFIC_ALL                                                                                         \
-    (H5F_RFIC_UNUSUAL_NUM_UNUSED_NUMERIC_BITS) /**< Suppress all format integrity check errors.  See         \
-                                                *   documentation for H5Pset_relax_file_integrity_checks     \
-                                                *   for details. */
+    (H5F_RFIC_UNUSUAL_NUM_UNUSED_NUMERIC_BITS) /**< Suppress all format integrity check                      \
+                                                * errors.  See documentation for                             \
+                                                * H5Pset_relax_file_integrity_checks                         \
```

Comment:
add () for linking:

H5Pset_relax_file_integrity_checks()

Reply:
I will fix this in my next PR.  Thanks.